site/src/routes/repl/[id]/index.json.js

```diff
@@ -0,0 +1,135 @@
+import send from '@polka/send';
+import redirect from '@polka/redirect';
+import body from '../_utils/body.js';
+import * as httpie from 'httpie';
+import { query, find } from '../../../utils/db';
+import { isUser } from '../../../backend/auth';
+import { get_example } from '../../examples/_examples.js';
+
+const { GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET } = process.env;
+
+async function import_gist(req, res) {
+	const base = `https://api.github.com/gists/${req.params.id}`;
+	const url = `${base}?client_id=${GITHUB_CLIENT_ID}&client_secret=${GITHUB_CLIENT_SECRET}`;
+
+	try {
+		const { data } = await httpie.get(url, {
+			headers: {
+				'User-Agent': 'https://svelte.dev'
+			}
+		});
+
+		// create owner if necessary...
+		let user = await find(`select * from users where uid = $1`, [data.owner.id]);
+
+		if (!user) {
+			const { id, name, login, avatar_url } = data.owner;
+
+			[user] = await query(`
+				insert into users(uid, name, username, avatar)
+				values ($1, $2, $3, $4) on conflict (uid) do nothing
```

Comment:
There'd be no conflict since you already checked for a user by its `uid` value

Reply:
ah, yep — leftover code. good catch